app/routes/routes.json

```
@@ -13,7 +14,7 @@
 	{
 		"description": "Return a random quote from \"Haddock\", The great captain in 'Les aventures de Tintin' wrote by Hergé.",
 		"endpoint": "haddock",
-		"quotesfile": "haddock",
-		"source": "Haddock"
+		"quotesfile": "Haddock",
```

The filename of Haddock's injures is `haddock.json` with `H` in lowercase.
The API "framework" use this exact string (case included) to automatically retrieve the JSON for the API.

So i suggest revert this :
```suggestion
		"quotesfile": "haddock",
```